content/reference-ledger-format.md

```diff
@@ -358,6 +362,118 @@ The ID of an Escrow object is the SHA-512Half of the following values put togeth
 * The AccountID of the sender of the [EscrowCreate transaction][] that created the Escrow object
 * The Sequence number of the [EscrowCreate transaction][] that created the Escrow object
 
+## FeeSettings
+[[Source]<br>](https://github.com/ripple/rippled/blob/master/src/ripple/protocol/impl/LedgerFormats.cpp#L115-L120 "Source")
+
+The `FeeSettings` object type contains the current base [transaction cost](concept-transaction-cost.html) and [reserve amounts](concept-reserves.html) as determined by [fee voting](concept-fee-voting.html). Each ledger version contains **at most one** `FeeSettings` object.
+
+Example `FeeSettings` object:
+
+```json
+{
+   "BaseFee": "000000000000000A",
+   "Flags": 0,
+   "LedgerEntryType": "FeeSettings",
+   "ReferenceFeeUnits": 10,
+   "ReserveBase": 20000000,
+   "ReserveIncrement": 5000000,
+   "index": "4BC50C9B0D8515D3EAAE1E74B29A95804346C491EE1A95BF25E4AAB854A6A651"
+}
+```
+
+The `FeeSettings` object has the following fields:
+
+| Name                | JSON Type | [Internal Type][] | Description            |
+|:--------------------|:----------|:------------------|:-----------------------|
+| `LedgerEntryType`   | String    | UInt16            | The value `0x0073`, mapped to the string `FeeSettings`, indicates that this object is a list of ledger hashes. |
```

Comment:
The text here is not correct. Should read "indicates that this object contains the network fee settings" or something like that.